templates/lib/adapter-config.d.ts.ts

```diff
@@ -7,7 +24,22 @@ const templateFunction: TemplateFunction = answers => {
 
 	const useTypeScript = answers.language === "TypeScript";
 	const useTypeChecking = answers.tools && answers.tools.indexOf("type checking") > -1;
-	if (useTypeScript && !useTypeChecking) return; // Don't do the copy/delete stuff in the first version... We'll add this later
+	if (useTypeScript && !useTypeChecking) {
+		const adapterSettings: AdapterSettings[] = answers.adapterSettings ?? getDefaultAnswer("adapterSettings")!;
+		
+		const template = `
```

Comment:
defining `template` twice looks weird. This would be better
```ts
let template: string;
if (...) {
    template = "...";
} else {
    template = "...";
}

return template.trim();
```

Reply:
Done.